python/llm/example/redpajama_simple.py

```diff
@@ -0,0 +1,5 @@
+from bigdl.llm.ggml.model import Gptneox
```

Comment:
Change to
```python
from bigdl.llm.models import GPTNeoX
```

Reply:
- This new API is not ready yet. Once its ready I'll change it in langchain integrations instead. 
- These simple examples (e.g. `llama_simple` and `redpajama_simple` ) are just for validation purposes (langchain users won't use these llm APIs directly). I plan to make them into UTs once CI/CD is ready.  

@TheaperDeng please inform people who's working on examples to also use the new interface `bigdl.llm.models`.